README.md

```diff
@@ -12,25 +12,56 @@ Python 3 bindings for the AWS Common Runtime.
 This library is licensed under the Apache 2.0 License.
 
 ## Minimum Requirements:
+
 *   Python 3.7+
 
 ## Installation
 
 To install from pip:
-````bash
+
+```bash
 python3 -m pip install awscrt
-````
+```
 
 To install from Github:
-````bash
+
+```bash
 git clone https://github.com/awslabs/aws-crt-python.git
 cd aws-crt-python
 git submodule update --init
 python3 -m pip install .
-````
+```
 
 To use from your Python application, declare `awscrt` as a dependency in your `setup.py` file.
 
+### OpenSSL and LibCrypto (Unix only)
+
+aws-crt-python does not use OpenSSL for TLS.
+On Apple and Windows devices, the OS's default TLS library is used.
+On Unix devices, [s2n-tls](https://github.com/aws/s2n-tls) is used.
+But s2n-tls uses libcrypto, the cryptography math library bundled with OpenSSL.
+
+You can ignore all this on Windows and Apple platforms, where aws-crt-python
+uses the OS's default libraries for TLS and cryptography math.
+
```

Comment:
Oops, this sentence is in here twice. Once here, and again at the end of the section. let's remove this one. I edit things to death, and sometimes leave in ghosts of old edits by accident
```suggestion
```